### PR TITLE
Tabular: Avoid showing plot by default in fit_summary()

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*
#1062 

*Description of changes:*

- Disables showing plot by default to avoid halting scripts, particularly if the user is using command line.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by gradientsky — APPROVED** [review]

**Comment by szha** [comment]
Job PR-1166-2 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1166/2/index.html

File: tabular/src/autogluon/tabular/predictor/predictor.py [hunk 1]
```diff
@@ -1085,17 +1085,19 @@ def leaderboard(self, data=None, extra_info=False, extra_metrics=None, only_pare
...
 
         Parameters
         ----------
         verbosity : int, default = 3
-            Controls how detailed of a summary to ouput.
+            Controls how detailed of a summary to output.
             Set <= 0 for no output printing, 1 to print just high-level summary,
             2 to print summary and create plots, >= 3 to print all information produced during `fit()`.
+        show_plot : bool, default = False
+            If True, shows the plot in browser when verbosity > 1.
```
Comment (jwmueller) [inline tabular/src/autogluon/tabular/predictor/predictor.py:1100]: could clarify: "plot" -> "model summary plot"

**Review by jwmueller — APPROVED** [review]
Looks good, but show_plot should be set = True in the tutorials right?
Otherwise most users would probably not even know these plots exist.

**Comment by Innixma** [comment]
> Looks good, but show_plot should be set = True in the tutorials right?
> Otherwise most users would probably not even know these plots exist.

Currently they don't show in tutorials anyways because bokeh isn't installed.

If you'd like you could send a PR which enables them in tutorials and visualizes them in our documentation.

**Comment by jwmueller** [comment]
> > Looks good, but show_plot should be set = True in the tutorials right?
> > Otherwise most users would probably not even know these plots exist.
> 
> Currently they don't show in tutorials anyways because bokeh isn't installed.
> 
> If you'd like you could send a PR which enables them in tutorials and visualizes them in our documentation.

The bokeh plot would not show on website regardless because it is in new browser, but would show for user who runs the tutorial locally in interactive python/notebook with bokeh installed (as should the other plots).  One option is the tutorial has: show_plot = True with a comment to set this =False to avoid halting in non-interactive Python scripts.  Or tutorials can just have plots disabled (I'm just noting then we can't expect most users will ever see plots)

**Comment by szha** [comment]
Job PR-1166-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1166/3/index.html

File: tabular/src/autogluon/tabular/predictor/predictor.py (hunk 1 above)
Comment (Innixma) [reply tabular/src/autogluon/tabular/predictor/predictor.py]: added

**Comment by Innixma** [comment]
Added `show_plot=True` in tutorials, although it will look a bit odd since no plots will be shown on the website.

**Comment by szha** [comment]
Job PR-1166-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-1166/4/index.html

**Pull request merged**